Makefile

```diff
@@ -100,28 +85,17 @@ releases:
 package: clean releases
 
 .PHONY: sync
-sync: 
+sync:
 	scripts/charm_helpers_sync.py -d hooks/charmhelpers -c charm-helpers.yaml
 
-.PHONY: publish-yellow
-publish-yellow: clean
-	$(CHARMCMD) upload --publish . $(YELLOW_CS_URL)
-
-.PHONY: publish-promulgated
-publish-promulgated: clean
-	$(CHARMCMD) upload --publish . $(PROMULGATED_CS_URL)
-
```

Comment:
I would like for these targets to be fixed rather than removed.  Publishing is error prone and mechanizing it is preferred over documenting which is preferred over our tendency towards oral history.